src/main/java/org/broadinstitute/hellbender/tools/spark/sv/ContigAligner.java

```diff
@@ -0,0 +1,272 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
+import com.github.lindenb.jbwa.jni.AlnRgn;
+import com.github.lindenb.jbwa.jni.BwaIndex;
+import com.github.lindenb.jbwa.jni.BwaMem;
+import com.github.lindenb.jbwa.jni.ShortRead;
+import htsjdk.samtools.Cigar;
+import htsjdk.samtools.CigarOperator;
+import htsjdk.samtools.TextCigarCodec;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.bwa.BWANativeLibrary;
+import scala.Tuple2;
+
+import java.io.Closeable;
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ContigAligner implements Closeable {
+
+    final BwaIndex index;
+    final BwaMem bwaMem;
+
+    public ContigAligner(final File referenceFile) {
+        BWANativeLibrary.load();
+        try {
+            index = new BwaIndex(referenceFile);
+        } catch (IOException e) {
+            throw new GATKException("Unable to load reference", e);
+        }
+        bwaMem = new BwaMem(index);
+
+    }
+
+    public List<AssembledBreakpoint> alignContigs (final RunSGAViaProcessBuilderOnSpark.ContigsCollection contigsCollection) {
+        final List<AssembledBreakpoint> assembledBreakpoints = new ArrayList<>();
+
+
+            try {
+                for(final Tuple2<RunSGAViaProcessBuilderOnSpark.ContigsCollection.ContigID, RunSGAViaProcessBuilderOnSpark.ContigsCollection.ContigSequence> contigInfo : contigsCollection.getContents()) {
+                    final String contigId = contigInfo._1.toString();
+                    final byte[] sequence = contigInfo._2.toString().getBytes();
+                    final AlnRgn[] alnRgns = bwaAlignSequence(bwaMem, contigId, sequence);
+
+                    // todo: parse cigar for internal indels
```

Comment:
There's also the issue about the assembler not necessarily putting together a nice chimeric contig to consider.  (We'd have to explore the graph connectivity info to understand these.)  Is it an issue?  Or do we know that SGA will output chimeras when possible?  File it as an issue to investigate?